### ISSUE TITLE
test_po_update should skip for asic with no portchannel

### ISSUE THREAD

**Issue by anamehra** [sonic-net/sonic-mgmt#7314]
**Description**

test_po_update fails for asic with no portchannel on multi-asic system:
    def test_po_update(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_frontend_asic_index, tbinfo):
        &quot;&quot;&quot;
        test port channel add/deletion as well ip address configuration
        &quot;&quot;&quot;
        duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
        asichost = duthost.asic_instance(enum_frontend_asic_index)
        int_facts = asichost.interface_facts()[&apos;ansible_facts&apos;]

        port_channels_data = asichost.get_portchannels_and_members_in_ns(tbinfo)
        portchannel = None
        portchannel_members = None
        for portchannel in port_channels_data:
            logging.info(&apos;Trying to get PortChannel: {} for test&apos;.format(portchannel))
            if int_facts[&apos;ansible_interface_facts&apos;][portchannel].get(&apos;ipv4&apos;):
                portchannel_members = port_channels_data[portchannel]
                break

&gt;       pytest_assert(portchannel and portchannel_members, &apos;Can not get PortChannel interface for test&apos;)
E       Failed: Can not get PortChannel interface for test

**Steps to reproduce the issue:**
1.
2.
3.

**Describe the results you received:**
Test case failed for the asic with no portchannel

**Describe the results you expected:**
The test should skip the asic with no portchannel

**Additional information you deem important:**

    **Output of `show version`:**

    ```
Using sonic-mgmt 202205 sha https://github.com/sonic-net/sonic-mgmt/commit/15d9388cebe1ad487656576647c69ade26df0d0a
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

**Comment by anamehra** [comment]
@abdosi , @vperumal , FYI.

**Comment by yxieca** [comment]
@anamehra  this issue is a test condition issue right? from my understanding, There is no ASIC support or SONiC image issue involved, it is just that there are DUT where no lag was configured?

**Comment by anamehra** [comment]
Fix for this was merged via #7574
